src/plugin.js

```diff
@@ -2,6 +2,20 @@ import videojs from 'video.js';
 import window from 'global/window';
 import document from 'global/document';
 
+/** THe logic below is used to check if flash is disabled or not installed
+ *  in IE browser and show the appropriate message to the user.
+ */
+let isSupported = window.navigator.mimeTypes['application/x-shockwave-flash'];
```

Comment:
why not [ask videojs whether flash is supported](https://github.com/videojs/video.js/blob/master/src/js/tech/flash.js#L365-L368)?
```js
const isFlashSupported = videojs.getComponent('Flash').isSupported();
```